addons/mail/static/src/discuss/thread_model_patch.js

```diff
@@ -15,7 +15,7 @@ patch(Thread.prototype, "discuss/core/common", {
             );
         }
         if (this.type === "chat") {
-            return `/web/image/res.partner/${this.chatPartnerId}/avatar_128`;
+            return `/web/image/res.partner/${this.chatPartnerId}/avatar_128?unique=${this.chatPartnerWriteDate}`;
```

Comment:
Shouldn't this use `avatarUrl` with persona?

Reply:
Except that you don't have a persona in `imgUrl`, and can't access `avatarUrl` since it's from a service `imgUrl` is inside a model.
I could store the whole persona instead of just the write date, but that feel overkill for a fix. Since in master the correct fix would be to transform `chatPartnerId` in `chatPersona` and use that.

Reply:
You have `this.chatPartnerId` and `this._store` this is enough to find any persona you need.

Reply:
About the access, it means places where `imgUrl` is used are incorrect, and should be replaced by `avatarUrl` up in the chain until where the service is accessible.

Reply:
I guess `avatarUrl` issue here can be ignored here in the scope of this PR, but the write date should be retrieved from the person at least.